README.md

```diff
@@ -13,72 +14,115 @@
   <h1>PostCSS CLI</h1>
 </div>
 
-## Installation
+<h2 align="center">Install</h2>
 
 ```bash
-npm i -D postcss-cli
+npm i -g|-D postcss-cli
 ```
 
-## Usage
+<h2 align="center">Usage</h2>
 
+```bash
+postcss [input.css] [OPTIONS] [-o|--output output.css] [-w|--watch]
 ```
-postcss [input.css] [OPTIONS] [--output|-o output.css] [--watch]
-
-Options:
-  -o, --output       Output file                                        [string]
-  -d, --dir          Output directory                                   [string]
-  -r, --replace      Replace (overwrite) the input file                [boolean]
-  -u, --use          List of postcss plugins to use                      [array]
-  -p, --parser       Custom postcss parser                              [string]
-  -t, --stringifier  Custom postcss stringifier                         [string]
-  -s, --syntax       Custom postcss syntax                              [string]
-  -w, --watch        Watch files for changes and recompile as needed   [boolean]
-  -x, --extension    Override the output file extension                 [string]
-  -e, --env          A shortcut for setting NODE_ENV                    [string]
-  -c, --config       Set a custom path to look for a config file        [string]
-  -m, --map          Create an external sourcemap
-  --no-map           Disable the default inline sourcemaps
-  -h, --help         Show help                                         [boolean]
-  -v, --version      Show version number                               [boolean]
-
-Examples:
-  postcss input.css -o output.css       Basic usage
-  cat input.css | postcss -u            Piping input & output
-  autoprefixer > output.css
-
-If no input files are passed, it reads from stdin. If neither -o, --dir, or
---replace is passed, it writes to stdout.
 
-If there are multiple input files, the --dir or --replace option must be passed.
+> ⚠️  If there are multiple input files, the --dir or --replace option must be passed.
+
+```bash
+cat input.css | postcss [OPTIONS] > output.css
 ```
 
-For more details on custom parsers, stringifiers and syntaxes, see the [postcss docs](https://github.com/postcss/postcss#syntaxes).
+> ⚠️  If no input files are passed, it reads from stdin. If neither -o, --dir, or
+--replace is passed, it writes to stdout.
+
+<h2 align="center">Options</h2>
+
+|Name|Type|Default|Description|
+|:---|:--:|:-----:|:----------|
+|`-d, --dir`|`{String}`|`undefined`|Output Directory|
+|`-o, --output`|`{String}`|`undefined`|Output File|
+|`-r, --replace`|`{String}`|`undefined`|Input <=> Output|
```

Comment:
Perhaps `Input <=> Output` could be expanded to something a bit more descriptive.